ocs_ci/ocs/constants.py

```diff
@@ -1904,13 +1904,13 @@
         "vg-manager_label": "app=vg-manager",
     },
     "default": {
-        "controller_manager_label": "app.kubernetes.io/name=lvm-operator",
-        "topolvm-controller_label": "app.kubernetes.io/name=topolvm-controller",
-        "topolvm-node_label": "app.kubernetes.io/name=topolvm-node",
+        "controller_manager_label": "app.kubernetes.io/name=lvms-operator",
+        "topolvm-controller_label": "app.kubernetes.io/name=topolvm-csi-driver",
+        "topolvm-node_label": "app.kubernetes.io/name=topolvm-csi-driver",
```

Comment:
The two labels above are the same so then can be a confusion.
You can use the component which is unique for both of them.
app.kubernetes.io/component=topolvm-controller
app.kubernetes.io/component=topolvm-node